services/guard/api.go

```diff
@@ -0,0 +1,36 @@
+package guard
+
+import (
+	"errors"
+
+	"github.com/dedis/cothority/log"
+	"github.com/dedis/cothority/network"
+	"github.com/dedis/cothority/sda"
+	"github.com/dedis/crypto/abstract"
+)
+
+// Client is a structure to communicate with Guard service
+type Client struct {
+	*sda.Client
+}
+
+// NewClient makes a new Client
+func NewClient() *Client {
+	return &Client{Client: sda.NewClient(ServiceName)}
+}
+
+// GetGuard is the function that sends a request to the guard server and creates a client to receive the responses
```

Comment:
Update comment to reflect what is really happening here. (client sends message ...)